### PR TITLE
(#135) Add option to control repo GPG checking

### PR DESCRIPTION
**Pull request by legooolas**
This adds the optional 'repo_gpgcheck' boolean option, for the added yum repo.
This isn't possible to use with packagecloud repo mirrors, so this is
kept as disabled by default to ensure consistent behaviour with
existing users.

Backward compatibility is retained.

### CODE REVIEW

**Comment by legooolas** [comment]
Does this look OK?

File: manifests/repo.pp [hunk 1]
```diff
@@ -19,7 +20,7 @@
       ensure          => $ensure,
       descr           => 'Choria Orchestrator Releases',
       baseurl         => "${choria::repo_baseurl}/release/el/${release}/\$basearch",
-      repo_gpgcheck   => false,
+      repo_gpgcheck   => $repo_gpgcheck,
```
Comment (ripienaar) [inline manifests/repo.pp]: can simply and just use `$choria::repo_gpgcheck` which is in line with how `$choria::repo_baseurl` work etc, same pattern

Comment (legooolas) [reply manifests/repo.pp]: Updated to use this instead.
(Sorry for the very long delay before changing this!)

**Comment by ripienaar** [comment]
cool cool - mind rebasing to master and squasing to one commit? thanks!

**Comment by ripienaar** [comment]
that merge was probably a bad idea, now you wont get back to a single commit :)

**Comment by legooolas** [comment]
Fixed with some resetting and a force push to my PR branch  :)
Also set the param to the choria class as optional, as I forgot to do that before.

**Comment by ripienaar** [comment]
thank you :)

**Pull request merged**